_unittest/conftest.py

```diff
@@ -172,14 +182,14 @@ def setup(self):
 
 @pytest.fixture(scope="session", autouse=True)
 def desktop_init():
-    desktop = Desktop(desktop_version, non_graphical, new_thread)
-    desktop.disable_autosave()
-    yield desktop
-
+    # desktop = Desktop(desktop_version, non_graphical, new_thread)
+    # desktop.disable_autosave()
+    # yield desktop
+    yield
     # If new_thread is set to false by a local_config, then don't close the desktop.
     # Intended for local debugging purposes only
-    if new_thread or os.name == "posix":
-        desktop.close_desktop()
+    # if new_thread or os.name == "posix":
+    #    desktop.close_desktop()
```

Comment:
@maxcapodi78 It is not required anymore to have this new-thread for local debugging session?
```suggestion
```

Reply:
Thanks